--- git status ---
HEAD detached at v2.3.0
Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git restore <file>..." to discard changes in working directory)
	modified:   source/isaaclab_tasks/isaaclab_tasks/manager_based/manipulation/__init__.py
	modified:   source/isaaclab_tasks/isaaclab_tasks/manager_based/manipulation/lift/__init__.py
	modified:   source/isaaclab_tasks/isaaclab_tasks/manager_based/manipulation/lift/lift_env_cfg.py

Untracked files:
  (use "git add <file>..." to include in what will be committed)
	source/isaaclab_tasks/isaaclab_tasks/manager_based/manipulation/lift/config/ur5_robotiq/
	source/isaaclab_tasks/isaaclab_tasks/manager_based/manipulation/ur5_robotiq_stack/

no changes added to commit (use "git add" and/or "git commit -a") 


--- git diff ---
diff --git a/source/isaaclab_tasks/isaaclab_tasks/manager_based/manipulation/__init__.py b/source/isaaclab_tasks/isaaclab_tasks/manager_based/manipulation/__init__.py
index 8f5703b9b32..f67643f83f8 100644
--- a/source/isaaclab_tasks/isaaclab_tasks/manager_based/manipulation/__init__.py
+++ b/source/isaaclab_tasks/isaaclab_tasks/manager_based/manipulation/__init__.py
@@ -6,3 +6,4 @@
 """Manipulation environments for fixed-arm robots."""
 
 from .reach import *  # noqa
+from .lift import *  # noqa
diff --git a/source/isaaclab_tasks/isaaclab_tasks/manager_based/manipulation/lift/__init__.py b/source/isaaclab_tasks/isaaclab_tasks/manager_based/manipulation/lift/__init__.py
index 2cec57c1cc9..7d564ef87e4 100644
--- a/source/isaaclab_tasks/isaaclab_tasks/manager_based/manipulation/lift/__init__.py
+++ b/source/isaaclab_tasks/isaaclab_tasks/manager_based/manipulation/lift/__init__.py
@@ -1,9 +1,5 @@
-# Copyright (c) 2022-2025, The Isaac Lab Project Developers (https://github.com/isaac-sim/IsaacLab/blob/main/CONTRIBUTORS.md).
-# All rights reserved.
-#
+# Copyright (c) 2022-2025, The Isaac Lab Project Developers.
 # SPDX-License-Identifier: BSD-3-Clause
-
-"""Configurations for the object lift environments."""
-
-# We leave this file empty since we don't want to expose any configs in this package directly.
-# We still need this file to import the "config" module in the parent package.
+"""Lift environments for fixed-arm robots."""
+from .config.franka import *  # noqa
+from .config.ur5_robotiq import *  # noqa
diff --git a/source/isaaclab_tasks/isaaclab_tasks/manager_based/manipulation/lift/lift_env_cfg.py b/source/isaaclab_tasks/isaaclab_tasks/manager_based/manipulation/lift/lift_env_cfg.py
index 3a4f458854d..33b1c00c337 100644
--- a/source/isaaclab_tasks/isaaclab_tasks/manager_based/manipulation/lift/lift_env_cfg.py
+++ b/source/isaaclab_tasks/isaaclab_tasks/manager_based/manipulation/lift/lift_env_cfg.py
@@ -177,11 +177,11 @@ class CurriculumCfg:
     """Curriculum terms for the MDP."""
 
     action_rate = CurrTerm(
-        func=mdp.modify_reward_weight, params={"term_name": "action_rate", "weight": -1e-1, "num_steps": 10000}
+        func=mdp.modify_reward_weight, params={"term_name": "action_rate", "weight": -1e-1, "num_steps": 99999999}
     )
 
     joint_vel = CurrTerm(
-        func=mdp.modify_reward_weight, params={"term_name": "joint_vel", "weight": -1e-1, "num_steps": 10000}
+        func=mdp.modify_reward_weight, params={"term_name": "joint_vel", "weight": -1e-1, "num_steps": 99999999}
     )
 
 